examples/components/ScrollView/ScrollView.example

```diff
@@ -0,0 +1,15 @@
+class Example extends React.Component {
+
+  render() {
+    return (
+      <div style={{ height: 300 }}>
+        <ScrollView>
+          <div style={{ height: 600}}>
+            <img alt='scroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scrollscroll scroll' style={{ width: '100%'}} src='https://media.giphy.com/media/26BRERwHtgJTf7rTG/giphy.gif' />
```

Comment:
i put a picture (remote url) in the scrollable area and this would be the alternative text if there's no internet. The picture can be considered spammy too

Do you like it? 
![](https://media.giphy.com/media/26BRERwHtgJTf7rTG/giphy.gif)